client/jetpack-connect/cookies.js

```diff
@@ -0,0 +1,26 @@
+/** @format */
```

Comment:
Rather than `cookies` which describes implementation details, it might be better to call this something related to plans or persistence… `plan-persistence`?

Reply:
Yeah I thought you would say this. The thing is, the only reason we have this is because of the implementation details, so there is some sense to it. You're right though, let's call it by what it does, I'll change it.